gulpfile.js

```diff
@@ -20,13 +23,12 @@ var jekyllConfig = jekyllConfigs.LOCAL
 var sources = {
   content: 'app/**/*.{markdown,md,html,txt,yml,yaml}',
   styles: 'app/_assets/stylesheets/**/*.{less,css}',
-  js: 'app/_assets/javascripts/**/*.js',
+  js: ['app/_assets/javascripts/**/*.js', 'bower_components/bootstrap/js/dropdown.js'],
   images: 'app/_assets/images/**/*',
   fonts: 'bower_components/font-awesome/fonts/**/*.*'
 }
 
 gulp.task('styles', function () {
-  var glob = require('glob')
```

Comment:
Why changing the style of an established project just to make it to your own sauce? It was like that, it is nicer because those are local dependencies, respect it and move on.